### DISCUSSION TITLE
sliding-interface 的初筛

### DISCUSSION THREAD

**Discussion by Goldeninscriptions** [APSIS-ANALYSIS/PERIGEE#143] [Ideas]
在每个交界面上，将rotated单元划分到地球上的时区一样的片区，然后根据初始位置给fixed单元标定最近的时区。

在圆柱面上时区沿旋转轴切片，类似切片的虎皮蛋糕卷；在顶部和底部的圆面上时区按半径划分，时区类似射击靶上的环。

fixed单元上的点先从标定的时区里找对面单元上的对面点，本时区内找不到时再找（左/右 或 内/外）相邻的时区。不太可能找不到。

缺点是前处理的h5文件会有更多数据，优点是找对面单元的循环从更合理的地方开始。

**Comment by ju-liu** [comment]
我的想法是：可以将二维的曲面们作为独立的剖分单元，使用同三维剖分一样的方式进行前处理。这样我们可以将fixed的区域尽可能地压缩，使用强扩展来提高效率。当然，前处理也需要更新h5文件的数据结构。

可斟酌。
